tools/build.sh

```diff
@@ -0,0 +1,63 @@
+#! /bin/bash
+RELEASE_VERSION=$(cat version.txt | head -n1)
+# follow cmake PACKAGE_FILE_NAME directive in main repo
+RELEASE_STUB="OpenBangla-Keyboard_${RELEASE_VERSION}-"
+
+makeDeb () {
+    RELEASE_FILENAME="${RELEASE_STUB}${DIST}.deb"
+    apt-get -y install build-essential cmake libibus-1.0-dev qt5-default rustc cargo ninja-build curl
+    cmake -H"$GITHUB_WORKSPACE" -B/build -GNinja -DCPACK_GENERATOR=DEB
+    ninja package -C /build
+    RELEASE_FILE="/build/${RELEASE_FILENAME}"
+}
+
+makeRpm () {
+    RELEASE_FILENAME="${RELEASE_STUB}${DIST}.rpm"
+    dnf install -y --allowerasing @buildsys-build cmake ibus-devel qt5-qtdeclarative-devel rust cargo ninja-build
+    cmake -H"$GITHUB_WORKSPACE" -B/build -GNinja -DCPACK_GENERATOR=RPM
+    ninja package -C /build
+    RELEASE_FILE="/build/${RELEASE_FILENAME}"
+}
+
+makeArch () {
+    DIST=archlinux
+    # pin package extension
+    PKGEXT=".pkg.tar.xz"
```

Comment:
Why we should package with the old compression algorithm? [ZStd is the default compression algorithm with the `.pkg.tar.zst` extension](https://www.archlinux.org/news/now-using-zstandard-instead-of-xz-for-package-compression/) with better choices.

Reply:
fixed.